h5bp/security/x-frame-options.conf

```diff
@@ -15,14 +15,17 @@
 #
 # Keep in mind that while you could send the `X-Frame-Options` header for all
 # of your website's pages, this has the potential downside that it forbids even
-# non-malicious framing of your content (e.g.: when users visit your website
-# using a Google Image Search results page).
+# non-malicious framing of your content
 #
 # Nonetheless, you should ensure that you send the `X-Frame-Options` header for
 # all pages that allow a user to make a state-changing operation (e.g: pages
 # that contain one-click purchase links, checkout or bank-transfer confirmation
 # pages, pages that make permanent configuration changes, etc.).
 #
+
+# Note: The Content-Security-Policy HTTP header has a frame-ancestors directive
+# which obsoletes this header for supporting browsers.
```

Comment:
Not in the issue #275, but it could be cool to add this info copied from https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options